src/components/StoreContainer.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Categories from './Categories';
+import BookList from './BookList';
+import Header from './Header';
+
+class Container extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      books: [
+        {
+          title: 'The hunger Games',
+          author: 'Suzanne Collins',
+          id: 1,
+        },
+        {
+          title: 'The Dune',
+          author: 'Frank Herbert',
+          id: 2,
+        },
+        {
+          title: 'Capital in the Twenty-First Century',
+          author: 'Suzanne Collins',
+          id: 3,
+
+        },
+      ],
+    };
+  }
+
+  delBooks = () => {
+    console.log('remove books');
+  };
+
+  render() {
+    const { books } = this.state;
+    return (
+      <div>
+        <Header />
+        <Routes>
+          <Route exact path="/" element={<BookList books={books} deleteBooksProps={this.delBooks} />} />
+          <Route path="categories" element={<Categories />} />
+        </Routes>
+      </div>
+    );
+  }
+}
+
```

Comment:
- [ ] I got a plain white background when the page loads. Can you please check what the problem might me? I'm thinking that the problem is coming from the `BookList` component. Because the index should run display the `BookList` and the `AddBook` components.

Reply:
The problem was the way I destructured my props actually. I fixed it and it should render the components appropriately now. Thank you for looking out.
![bookstore](https://user-images.githubusercontent.com/66526480/166084148-d2ced03b-cc27-49d7-b2f0-33a003881d7d.PNG)